descriptors/ModuleDescriptor-template.json

```diff
@@ -226,6 +226,30 @@
         }
       ]
     },
+    {
```

Comment:
srs does not provide this endpoint.
It must be added to "requires" section (check mod-search for reference)

Reply:
done